src/checkStrength.ts

```diff
@@ -0,0 +1,41 @@
+import Points from './Points';
+
+interface ICheckResult {
+  points: number;
+  range: string;
+}
+
+interface ICheckStrength {
+  (password: string, validSymbolsChars?: string): ICheckResult;
+}
+
+const getRange = (points: number) => {
```

Comment:
I think that is more cool to send the points in number and the client resolve how to use...

Reply:
I prefer to change because it’s leaner, without {}

Reply:
He already sends the points in number and with their respective classifications, it is easier to send the classification back instead of validating after the call

Reply:
Ahh Ok, sorry